MonoGame.Framework.Content.Pipeline/Builder/PipelineBuildEvent.cs

```diff
@@ -176,7 +185,7 @@ public bool NeedsRebuild(PipelineManager manager, PipelineBuildEvent cachedEvent
 
             // If the source file is newer than the dest file
             // then it must have been updated and needs a rebuild.
-            if (sourceWriteTime >= destWriteTime)
+            if (sourceWriteTime > destWriteTime)
```

Comment:
If looking at the code can't explain why the change was needed, it should have a simple comment stating why it is the way it is.